### ISSUE TITLE
Just reply no actual isu

### ISSUE THREAD

**Issue by DigitalCheese** [DVT7125/website#1]
Wait a minute aren't you The Tundra on Guilded? If so, tell me otherwise ignore this msg I might have you mixed up with someone else lol (btw if u are tundra, i dont use my outlook email anymore but rather the ATT one i have)

**Comment by DVT7125** [comment]
> Wait a minute aren't you The Tundra on Guilded? If so, tell me otherwise ignore this msg I might have you mixed up with someone else lol (btw if u are tundra, i dont use my outlook email anymore but rather the ATT one i have)

Yeah I'm the tundra / voidless. I'm good. :) (Email me for more info)

**Comment by DVT7125** [comment]
Yes I am. I'm at school lol, im good. I aint dead, and im off from the
drinking. :)

**Comment by DigitalCheese** [comment]
gud lol, also i cant reply directly via email despite it saying i can so that sucks but whatever lol. glad ur ok :]

**Comment by DVT7125** [comment]
> gud lol, also i cant reply directly via email despite it saying i can so that sucks but whatever lol. glad ur ok :]

Thx. Please tell notaussie I'm okay, as I'm at school and can't get to discord... (well I found an exploit I think lmao, could use replit, or disable the extension in android settings. I expect to be back soon, although my server power supplies are taken away, laptop, phone, even watch... and then for my birthday I got a new mouse and headset... 🤦

Old mouse: M720 (Logi) (1000 DPI, missing receiver, and connection 1 always on, drainingalkalinee battery to replace every week.)
New Mouse: Razer basilisk v3 pro (Up to 30k dpi, better thumb rest, and rgb (I don't care for it but lol) and a battery that lasts 2 days.... (cus its rechargeable, not alkaline where power varies with how long u have been using it.)

Old headset: H800 (Logi) 2H battery life, and 8000 hz microphone quality (Old telephone quality), and CD quality for sound (36k hz?)
New headset: Razer barracuda X, over 40h of battery life, 48000hz on sound and mic on bluetooth, (DVD), and Studio quality on receiver. (96000hz!)

I loved my old stuff too, my new headset is not open back, (aka it mutes out the sound around me), and my mouse has no "invis/silent" button to change windows.... 😢, But its still a GREAT upgrade. (and I can crash the chromebook with my mouse, with this enabled
![image](https://github.com/DVT7125/website/assets/99568947/15ec132a-a9c6-4bfd-81ce-8b401406d81e)

 Life is better tho, I get to do lots of fun things... (even if lots of stuff is taken away for now)

Thanks for checking in on me, talk soon! 🫡

(Reply to this if u want to ask any questions)
